lms/urls.py

```diff
@@ -61,6 +61,9 @@
 
     url(r'^heartbeat$', include('heartbeat.urls')),
 
+    # TODO jimabramson: put this in the right place
+    url(r'^api/profile_images/', include('openedx.core.djangoapps.profile_images.urls')),
```

Comment:
I would vote for namespacing in the new user API. What do you think, @andy-armstrong?